.evergreen/config.yml

```diff
@@ -419,16 +439,20 @@ functions:
 pre:
   - func: fetch source
   - func: prepare resources
+  - func: windows fix
   - func: fix absolute paths
   - func: make files executable
 post:
   - func: upload test results
   - func: cleanup
+ignore:
+  - '*.md'
```

Comment:
Very nice feature! Maybe we want to also enable it on the etc directory although maybe that's entering the danger zone.

Reply:
Sure I can add that in, though I was mostly thinking about doc fixes, which are fairly common.

Reply:
Yea true, you can leave it alone, there also could be something in the etc directory that impacts code one day somehow so better to leave it be.